src/com/pentalog/pentastagiu/homework2/shop/Basket.java

```diff
@@ -0,0 +1,55 @@
+package com.pentalog.pentastagiu.homework2.shop;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket {
+    private Customer customer;
+    private List<Product> product = new ArrayList<>();
+    private double totalPrice = 0;
+
+    public List<Product> getProduct() {
+        return product;
+    }
+
+    public void basketTotal(double price){
```

Comment:
foloseste verbe in denumirea metodelor: getTotalPrice sau calculatePriceForBasket, e mai usor de inteles de cei care iti vor citi codul si de fapt mai potrivita denumirea e de updateTotalPrice, de vreme ce invoici metoda asta de fiecare data dupa ce este adaugat un produs in cos
si accesorul e public, ceea ce inseamna ca oricine are acces si o poate apela si iti poate crea inconsistente in cos(iti updateaza pretul total al cosului fara sa adauge un produs), de aceea ar trebui setat private